health/health_config.c

```diff
@@ -650,6 +663,7 @@ static int health_readfile(const char *filename, void *data) {
             rt->name = strdupz(value);
             rt->hash_name = simple_hash(rt->name);
             rt->source = health_source_file(line, filename);
+            rt->edit_command = health_edit_command(line, filename);
```

Comment:
Hello @MrZammler ,

Please, add a call for `freez()` with this variable as argument inside `rrdcalctemplate_free()`, else we will have a memory leak when this PR is merged.

Thank you!

Reply:
Hi @thiagoftsm !

Thank you very much, I've pushed https://github.com/netdata/netdata/pull/10821/commits/d03606ba6e6a9d244c432dc515240d5e6c96a7f9